### PR TITLE
fix: add missing page titles

### LINKED ISSUE #929: Add missing page titles
This issue will track pages, whose titles are missing:

- [x] Job History Pages -> Will be fixed in #857
- [ ] Details Pages for Settings i.e. Settings -> Component -> Component Item
- [ ] Config Insights Page -> Configs -> Insights Tab

### PR DESCRIPTION
**Pull request by gitstart**
Closes #929

https://www.awesomescreenshot.com/video/15795111?key=3505faa3e8fccdc1ecd16402680d485e

---
This code was written and reviewed by GitStart Community. Growing great engineers, one PR at a time.

### CODE REVIEW

**Comment by gitstart** [comment]
@mainawycliffe this PR is ready for review

File: src/components/SchemaResourcePage/SchemaResourceEdit.tsx [hunk 1]
```diff
@@ -218,262 +219,276 @@ export function SchemaResourceEdit({
...
-                                        : undefined
-                                    }
-                                    onChange={(val) => {
-                                      setValueOnChange("labels", val);
-                                    }}
-                                    language="json"
+    <>
+      <Head
+        prefix={`Settings ${resourceName} - ${
+          subNav.find((nav) => nav.value === activeTab)?.label
```
Comment (mainawycliffe) [inline src/components/SchemaResourcePage/SchemaResourceEdit.tsx:225]: Don't use label here, use `name` prop, which is the name displayed in the name field

**Review by mainawycliffe — CHANGES_REQUESTED** [review]

File: src/components/SchemaResourcePage/SchemaResourceEdit.tsx (hunk 1 above)
Comment (gitstart) [reply src/components/SchemaResourcePage/SchemaResourceEdit.tsx]: Done @mainawycliffe, please check again.

**Review by mainawycliffe — APPROVED** [review]
LGTM

**Pull request merged**